helm/wind-river-cloud-platform-deployment-manager/templates/crds.yaml

```diff
@@ -1373,8 +1373,8 @@ spec:
                   location:
                     description: Location defines the physical location of the host in the data centre.
                     type: string
-                  maxCPUFrequency:
-                    description: MaxCPUFrequency defines the maximum limit of the CPU frequency set on the host.
+                  max_cpu_mhz_configured:
```

Comment:
Name mis-match

Reply:
Renamed to maxCPUMhzConfigured

Reply:
Since it is declared as max_cpu_mhz_configured [here ](https://github.com/Wind-River/gophercloud/pull/23/files#diff-17717160b479478faeca74dc8ec86bf1eb72eeb1801b047d6ccf301e528fbbc4R253), when we run deploy tool, it generates with this name automatically

Reply:
Please ignore my above comment. I renamed it to MaxCPUMhzConfigured